### PR TITLE
DOC-743 - [Activities] Add date filtering on activity browser calls

### PR DESCRIPTION
**Pull request by fbeaudoincoveo**
- Also documented the other parameters in the affected calls.
- Added some example values, but not complete example yet.

### CODE REVIEW

File: api_docs/Activity.yml [hunk 1]
```diff
@@ -180,45 +180,93 @@
...
+              The unique identifier of the organization from which to obtain entries.
             required: true
             type: "string"
             name: "organizationId"
             in: "path"
+            example: "mycoveocloudorganizationid"
           - 
-            description: "page"
+            description: |
+              The number of the "page" of entries to return in the `items` array of a successful response.
```
Comment (fdallaireCoveo) [inline api_docs/Activity.yml]: I would avoid quotes to highlight words ("page") and rather use the *italic* markup when defining a term, but here I would rephrase like: 
"The page number of paginated entries to return in the `items` array of a successful response."

File: api_docs/Activity.yml [hunk 2]
```diff
@@ -180,45 +180,93 @@
...
+
+              If you use the default value, the call returns all existing "pages".
```
Comment (fdallaireCoveo) [inline api_docs/Activity.yml]: No quotes on "pages" here and other occurrences.

File: api_docs/Activity.yml [hunk 3]
```diff
@@ -180,45 +180,93 @@
...
+
+              If you specify a "page" number that does not exist, the call returns an empty array (`[]`).
+
+              Otherwise, this parameter works in close relationship with the `perPage` parameter.
```
Comment (fdallaireCoveo) [inline api_docs/Activity.yml]: I would move this content up with the first sentence.

File: api_docs/Activity.yml [hunk 4]
```diff
@@ -180,45 +180,93 @@
...
             name: "page"
+            example: 10
           - 
-            description: "perPage"
+            description: |
+              The maximum number of entries per "page".
+
+              This parameter works in close relationship with the `page` parameter.
+
+              The value of the `totalPages` key in a successful response corresponds to the value of the `totalEntries`
```
Comment (fdallaireCoveo) [inline api_docs/Activity.yml]: Each time you write "of the" (French-like form), consider rather using the noun modifier form:
 "The value of the `totalPages` key in" becomes "The `totalPages` key value in" or even just "The `totalPages` value in". Do that everywhere possible.

File: api_docs/Activity.yml [hunk 5]
```diff
@@ -180,45 +180,93 @@
...
+              The number of the "page" of entries to return in the `items` array of a successful response.
+
+              If you use the default value, the call returns all existing "pages".
+
+              If you specify a "page" number that does not exist, the call returns an empty array (`[]`).
+
+              Otherwise, this parameter works in close relationship with the `perPage` parameter.
+
+              For example, if you set `page` to `3` and `perPage` to `25`, and the value of the `totalEntries` key in a
+              successful response is `100`, the `items` key will only contain the third "page" of entries (i.e.,
```
Comment (fdallaireCoveo) [inline api_docs/Activity.yml]: ...key only contains entries 51-75.

File: api_docs/Activity.yml [hunk 6]
```diff
@@ -1656,28 +1748,37 @@
...
           type: "integer"
           format: "int64"
+          example: 1491797963640
         endDate: 
           type: "integer"
           format: "int64"
+          example: 1491797963818
         resourceId: 
           type: "string"
+          example: "mycoveocloudorganization"
```
Comment (fdallaireCoveo) [inline api_docs/Activity.yml]: Is resourceId the organization name?

File: api_docs/Activity.yml [hunk 7]
```diff
@@ -1656,28 +1748,37 @@
...
         errorDetail: 
           type: "string"
+          example: null
         snapshotId: 
           type: "string"
+          example: null
         duration: 
           type: "integer"
           format: "int64"
+          example: 178
```
Comment (fdallaireCoveo) [inline api_docs/Activity.yml]: Where do we know what are the units?

File: api_docs/Activity.yml [hunk 8]
```diff
@@ -1740,21 +1849,28 @@
             - "PAUSED"
             - "EXECUTED"
           type: "string"
+          example: "EXECUTED"
         progress: 
           type: "number"
           format: "double"
+          example: 1
```
Comment (fdallaireCoveo) [inline api_docs/Activity.yml]: Where do we say what this number means?

File: api_docs/Activity.yml (hunk 6 above)
Comment (fbeaudoincoveo) [inline api_docs/Activity.yml]: In the case of the example I was crafting, yes (the example would show information about a backup activity, so the resourceId would actually be the organization id).

However, the organization id is not necessarily the best "generic" example value.

File: api_docs/Activity.yml (hunk 7 above)
Comment (fbeaudoincoveo) [inline api_docs/Activity.yml]: We don't... for now!

We simply need to document the response, but this is a matter for another JIRA :)

File: api_docs/Activity.yml (hunk 8 above)
Comment (fbeaudoincoveo) [inline api_docs/Activity.yml]: See my previous comment.
